ambry-api/src/main/java/com/github/ambry/clustermap/ReplicaStatusDelegate.java

```diff
@@ -46,6 +58,16 @@ public boolean unseal(ReplicaId replicaId) {
     return clusterParticipant.setReplicaSealedState(replicaId, false);
   }
 
+  /**
+   * Sets replicaId to unsealed status. Unsealed replicas are READ_WRITE for all blobs.
+   * @param replicaId the {@link ReplicaId} whose status would be set to read-only.
+   * @return {@code true} if replica is successfully sealed. {@code false} if not.
```

Comment:
nit: why set to "read-only"?
nit: s/sealed/unsealed

Reply:
fixed.